autoload/codefmt/formatterhelpers.vim

```diff
@@ -90,18 +90,21 @@ endfunction
 " @throws ShellError if the {cmd} system call fails
 " @throws WrongType
 function! codefmt#formatterhelpers#AttemptFakeRangeFormatting(
-    \ startline, endline, cmd) abort
+    \ startline, endline, cmd, ignoreerrors, skipfirstnlines) abort
```

Comment:
Can you make them optional using varargs, since they're kind of obscure options and almost always 0, 0?

Reply:
Done.